lib/proxy_api/proxy_exception.rb

```diff
@@ -3,12 +3,20 @@ class ProxyException < ::Foreman::WrappedException
     attr_reader :url
 
     def initialize(url, exception, message, *params)
+      message += append_exception_message(exception)
```

Comment:
Hmmm fair enough. I just realized after your comment that the error I did this for had to do with MAC addresses so it wouldn't be useful to have a different code per MAC :smile: I'll fix it, thanks @domcleal